packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -371,18 +376,14 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					{ alwaysCallNext: false }
 				);
 
+				viteDevServer.middlewares.use(middleware);
```

Comment:
Following on from my comment on the passthrough above, I don't believe we can do this here. See my other PR for how we think we an solve this more correctly (although still with potential dangers when interoperating with other plugins) - https://github.com/cloudflare/workers-sdk/pull/8494